### ISSUE TITLE
Filter quests containing a specific item

### ISSUE THREAD

**Issue by alexanderabramov** [Expugn/priconne-quest-helper#12]
As a player that wants to get a specific item/ingredient in the most efficient way,
I'd like to "load all projects" and then filter quest list only to quests with a chance for this specific item,
And see the quest list, as usual, sorted by total score for all projects and with other items from priority projects highlighted.

In terms of UI, I imagine this function can either be toggled on and off by clicking on the item in Items/Requested Items list, or via a new "filter" drop-down.

**Current alternative**
It is possible to scroll the very long list of all quests sorted by score and visually look for the quests with this item in main or additional drops.

**Comment by alexanderabramov** [comment]
For example, filter for "Emerald Earrings Fragment":
![image](https://user-images.githubusercontent.com/96938/62964215-dab4ba00-be02-11e9-97e9-e1839248ddba.png)
Whereas normally, these quests would be very far down the list

**Comment by Expugn** [comment]
I've added this feature in commit <https://github.com/Expugn/priconne-quest-helper/commit/08cf0bd216eb4eacae6bea000414151930cb83e3>.

To "focus" on an item, go to the `Items` tab and click on any item in any category. Any quest containing that item will then be displayed.

Focus can be switched by selecting on another item or removed if you click on the focused item.

Also, thanks for taking the time to write that pull request (<https://github.com/Expugn/priconne-quest-helper/pull/13>)! However, the implementation seemed too complicated so I ended up writing my own. Even though I ended up not using your code in the end it was still a great visualization on what you wanted!
